ROI_Maker.py

```diff
@@ -8,31 +7,31 @@ def ROI_Maker(spectrum, energy,sub_regions='auto'):
     """
     Takes in a measured spectrum and identifies peaks in terms of channel number
     and energy. Uses peak_finder to localize the peak more precisely from the
-    given energy. Also returns peak and compton ROIs as well as each ROIs gross
-    area.
+    given energy. Also returns peak and compton ROIs.
     """
 
     peak_energy = peak_finder(spectrum, energy)
 
     E0 = spectrum.energy_cal[0]
     Eslope = spectrum.energy_cal[1]
-    M_counts = spectrum.data
     energy_channel = int((peak_energy - E0) / Eslope)
 
     region_size = 1.3
     compton_distance = 4
 
     # Rough estimate of FWHM.
     fwhm = 0.05*peak_energy**0.5
-    fwhm_channel = int(region_size * (fwhm - E0) / Eslope)
-    # peak gross area
-    peak_ch = (energy_channel - fwhm_channel, energy_channel + fwhm_channel)
 
-    left_center = energy_channel - compton_distance * fwhm_channel
-    left_ch = (left_center - fwhm_channel, left_center + fwhm_channel)
+    region_width = int(region_size * (fwhm - E0) / Eslope)
+    region_separation = int(compton_distance*region_width)
```

Comment:
When I think "region separation", I think of the space between the right edge of the left Compton ROI, and the left edge of the peak ROI. In other words:
```
  left      peak      right
--|***|-----|***|-----|***|--
       <--->
      separation
```
Calculate what this separation is, in FWHM, for the existing code. (I mean, in your notebook or a whiteboard or whatever. Not in the python code.)

Then instead of using `compton_distance = 4` which doesn't help me visualize how separated the regions are, use `region_separation = ? * fwhm_ch` where `?` is a number.